test/auth.js

```diff
@@ -124,6 +124,14 @@ const main = () => {
     }
   })
 
+  test('[REST] cancelOpenOrders', async t => {
+    try {
+      await client.cancelOpenOrders({ symbol: 'ETHBTC'})
```

Comment:
Can you add the missing space before the `}`?

Reply:
done